step-plans/step-plans-core/src/main/java/step/core/artefacts/reports/ReportTreeAccessor.java

```diff
@@ -39,5 +39,14 @@ public interface ReportTreeAccessor {
 	 */
 	public Iterator<ReportNode> getChildren(String parentID);
 
+	/**
+	 * Returns the list of children of the ReportNode for a specific execution phase

```

Comment:
COmment not matching the new name "source"

Reply:
changed